### ISSUE TITLE
avoid https cert errors

### ISSUE THREAD

**Issue by Shuunen** [ericmdantas/aliv#34]
Hi :) 

The default ssl certs are invalid (invalid common name), personnally I use sslfie (https://github.com/mkropat/sslfie) to generate valid certificates, here is how I did to use aliv with valid local certs : 
for example lets say my local project is called "foobar", so I added `127.0.0.1  foobar.io` to my hosts file then in the `node_modules/aliv/lib/crt` folder I executed :
`sslfie -o server.crt -k server.key foobar.io`

To trust these new certificates : 
```
sudo apt install libnss3-tools
certutil -d sql:$HOME/.pki/nssdb -A -t P -n "Foobar" -i server.crt
``` 
That's it, after restarting browser, I can go to https://foobar.io:1307/ without having any https certificates errors :)

**Comment by ericmdantas** [comment]
Hey, @Shuunen.

I had never heard of `sslfie`, so thanks for the info.

When we first did the `https` thing, that was something that we knew would happen (the invalid common error), but we chose to let it go, since it wouldn't really bother the user too much. Having said that, I understand how it's preferred to have valid certs instead of the default one.

Is there any way you see this implemented? Something like a new option in the `.alivrc` file, like:

```json
//.alivrc

{
  "certPaths": {
      "ca": [
        "./my_cert/crt/server.csr"
      ],
      "cert": "./my_cert/crt/server.crt",
      "key": "./my_cert/crt/server.key"
  }
}
```

Or do you have something else in mind?

**Comment by Shuunen** [comment]
yes could be nice :)

is the csr really needed ? I have other projects running https servers without csr files

**Comment by ericmdantas** [comment]
It's one of the options Node gives you so you can inform your domain name, company name, etc. It might work without it, but it'd be up to the user to use it or not.

Edit: typo

**Comment by Shuunen** [comment]
sure :+1:

**Comment by ericmdantas** [comment]
Hey, @Shuunen.

I guess this PR https://github.com/ericmdantas/aliv/pull/38 solves this issue. It's released on `v1.10.0`.

Let me know how it goes.
